examples/ksml-runner.yaml

```diff
@@ -3,6 +3,7 @@ ksml:
   configDirectory: .              # When not set defaults to the working directory
   schemaDirectory: .              # When not set defaults to the config directory
   storageDirectory: /tmp          # When not set defaults to the default JVM temp directory
+  createStorageDirectory: false   # When set to true, the storage directory will be created if it does not exist
```

Comment:
Shall we keep the default to true since almost in all cases it will have to be enabled to avoid errors?

Reply:
I've made it false to not break the existing plain KSML images running, but enabled it by default in the Helm chart